FlowerShop/FlowerShop/Services/Network/AlamofireNetwork.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AlamofireNetwork.swift
+//  FlowerShop
+//
+//  Created by Andrei Oltean on 22/02/2020.
+//  Copyright © 2020 Andrei Oltean. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class AlamofireNetwork {
+    static var shared = AlamofireNetwork()
```

Comment:
Is this intended to be a singleton? if so, the "init" is not private and the class can be instantiated multiple times.